Source/Relativity.Test.Helpers/Relativity.Tests.Helpers.Tests.Unit/RetryHelper/RetryLogicHelperUnitTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Moq;
+using NUnit.Framework;
+using Relativity.API;
+using Relativity.OAuth2Client.Exceptions;
+using Relativity.Test.Helpers.Exceptions;
+using Relativity.Test.Helpers.Kepler;
+using Relativity.Test.Helpers.RetryHelper;
+using System;
+using System.Threading.Tasks;
+
+namespace Relativity.Tests.Helpers.Tests.Unit.RetryHelper
+{
+	[TestFixture]
+	public class RetryLogicHelperUnitTests
+	{
+		private Mock<IOAuth2Helper> _mockOAuth2Helper;
+		private Mock<IHelper> _mockHelper;
+
+		private RetryLogicHelper SuT;
+
+		private const string OAuth2Id = "OAuth2Id";
+		private const string OAuth2Secret = "OAuth2Secret";
+		private readonly Guid AppGuid = new Guid("20378139-2b25-4c8e-9c31-4aba4cc30162");

```

Comment:
for unit tests, I generally have dummy values to make it obvious. this guid looks like a real one.

Reply:
Updated this